beast/physicsmodel/creategrid.py

```diff
@@ -566,6 +570,9 @@ def add_spectral_properties(specgrid, filternames=None, filters=None,
         naming format to adopt for filternames and filters
         default value is '{0:s}_0' where the value will be the filter name
 
+    filterLib:  str
+        full filename to the filter library hd5 file
```

Comment:
This is the full *filename* of the filter library file.  This is needed for the travis testing as the library files are not in the source directory because the testing is done on an *installed* version of the beast.  This allows the testing code to directly pass the location of the filter library file instead of it being in the default location.

This raises the issue of how to handle the library files for the installed version of the BEAST (should make this for those just using the beast).  I've opened issue #141 for this.